x-pack/plugins/observability/public/context/date_picker_context.tsx

```diff
@@ -108,7 +117,7 @@ export function DatePickerContextProvider({ children }: { children: React.ReactE
   );
 
   useMount(() => {
-    updateUrl({ rangeFrom: relativeStart, rangeTo: relativeEnd });
+    updateUrl({ rangeFrom: relativeStart, rangeTo: relativeEnd }, true);
```

Comment:
nit: I personally try not to pass booleans as parameters because it isn't super clear when reading the function call as to what they mean.

One option would be to create another function like `updateUrlOnMount` that calls `updateUrl` with `true` or split out the functionality so that there are two separate functions that don't have the `if (isFirstRender)`.

Another option would be to make the `updateUrl` take an object as a parameter so that the boolean would have a named property like:

```
const updateUrl = useCallback(({ isFirstRender = false}: { ..., isFirstRender: boolean}) => { ... }
```

Reply:
Actually, that is not being passed anymore. I updated that logic after the first commit because I also didn't like it 😄 

 The `Files changed` didn't update immediately dunno why.